.github/workflows/ci.yaml

```diff
@@ -0,0 +1,139 @@
+name: CI
+on: [push, pull_request]
+jobs:
+    phpstan:
+        name: PHPStan
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php-version: [7.4]
```

Comment:
We should use the lowest supported version

Reply:
ok, I though most recent/fastest is ok here ...